sources/packages/backend/workflow/test/test-utils/mock/mock.utils.ts

```diff
@@ -20,32 +20,17 @@ const SERVICE_TASK_ID_SEPARATOR_REGEX = new RegExp(
 );
 
 /**
- * Get the mock identifier for a completed job to be sent to the workflow,
- * for instance, create_supporting_users_for_parents_task_result, where:
+ * Get the passthrough mock identifier for a completed job,
+ * for instance, create_supporting_users_for_parents_task_passthrough, where:
  * - `create_supporting_users_for_parents_task`: service task id;
- * - `result`: suffix that identifies the job completed object.
- * @param serviceTaskId service task id that will have the job completed returned.
- * @returns mock identifier for a completed job to be sent to the workflow.
+ * - `passthrough`: suffix that identifies the job passthrough identifier.
```

Comment:
`passthrough` is just a word change right, logically it is the same right. it resprest the passed/completed task/flow